src/recon_buildblock/BinNormalisationFromGEHDF5.cxx

```diff
@@ -310,13 +310,12 @@ read_norm_data(const string& filename)
     const int num_detectors_per_ring = scanner_ptr->get_num_detectors_per_ring();
 
     m_input_hdf5_sptr->initialise_efficiency_factors("");
-    std::array<unsigned long long int, 2> stride = {1, 1};
-    const std::array<unsigned long long int, 2> count  = {45, 448};
-    std::array<unsigned long long int, 2> offset = {0,0};
-    std::array<unsigned long long int, 2> block = {1, 1};
+
     unsigned int total_size = num_rings*num_detectors_per_ring;
     stir::Array<1, float> buffer(0, total_size-1);
-    m_input_hdf5_sptr->get_from_2d_dataset(offset, count, stride, block, buffer);
+    std::array<unsigned long long int, 2> stride = {1, 1};
+    std::array<unsigned long long int, 2> offset = {0, 0};
```

Comment:
could remove these as default?

Reply:
Yet error if they are not these values?

Reply:
Ah wait, you mean remove this, and add them as a default?

Reply:
yes, sorry. I thought you had them as default in some of these. If we have them everywhere, the "client" code becomes cleaner.